qa/zcash/performance-measurements.sh

```diff
@@ -134,6 +134,21 @@ function zcashd_valgrind_stop {
     cat valgrind.out
 }
 
+function zcashd_heaptrack_start {
+    rm -rf "$DATADIR"
+    mkdir -p "$DATADIR/regtest"
+    touch "$DATADIR/zcash.conf"
```

Comment:
This is incomplete because it is missing the `use_200k_benchmark` logic.

If we replace Massif with `heaptrack` per my previous comment, then just delete this new function and rename/modify `zcashd_massif_start` instead.